JitsiConference.js

```diff
@@ -18,6 +18,7 @@ var GlobalOnErrorHandler = require("./modules/util/GlobalOnErrorHandler");
 var JitsiConferenceEventManager = require("./JitsiConferenceEventManager");
 var VideoType = require('./service/RTC/VideoType');
 var Transcriber = require("./modules/transcription/transcriber");
+var TalkMutedDetection = require("./modules/talkmuted/TalkMutedDetection");
```

Comment:
I realize the following is not your fault because you created your PR before the introduction of ES6 in lib-jitsi-meet. However, I'd really like us to start taking advantage of ES6 features. Since rewriting the whole library in one go is unrealistic, I believe we could try to use ES6 in new PRs, especially ones adding new functionality. I've rewritten this PR using ES6 features and I'll push my modifications after publishing this note. The goal of this comment is to get my thoughts to you for future PRs.